src/main/java/oop/Stack.java

```diff
@@ -0,0 +1,33 @@
+
+package oop;
+
+public class Stack<T> {
+    private T[] stack;
+    private int count;
+
+    public Stack() {
+        stack = (T[]) new Object[100];
```

Comment:
```suggestion
        stack =  new T[100];
```

Reply:
Dạ nó không cho tạo trực tiếp như vậy ạ:
`Type parameter 'T' cannot be instantiated directly`
Vì nó không chắc T sẽ là Class, Abstract Class hay là Interface ạ.